test/client/Query.test.tsx

```diff
@@ -545,14 +549,18 @@ describe('Query component', () => {
     });
 
     it('default fetch-policy', done => {
+      let count = 0;
       const Component = () => (
         <Query query={allPeopleQuery}>
           {result => {
-            catchAsyncError(done, () => {
-              expect(result.loading).toBeFalsy();
-              expect(result.networkStatus).toBe(NetworkStatus.ready);
-              done();
-            });
+            if (count === 0) {
+              catchAsyncError(done, () => {
```

Comment:
Might want to go ahead and remove `catchAsyncError` usage, since we did that for 3.0?

Reply:
I think I'll leave this one as is, since these changes have to be merged into `release-3.0.0` by hand (and I'll make sure `catchAsyncError` isn't used). Thanks!